analytics_data_api/v0/views/learners.py

```diff
@@ -36,24 +36,33 @@ class LearnerView(CourseViewMixin, generics.RetrieveAPIView):
 
     **Response Values**
 
-        Returns the learner metadata and engagement data:
+        Returns metadata and engagement data for the learner in JSON format.
```

Comment:
Going back through several commits, you'll see that Peter asked what the format was, Dennis indicated that everything was JSON and so didn't particularly need to be called out as such, and then Peter pointed out that we did specifically say JSON format all the way down at line 280.
I'm happy to take all the JSON references out again, but think that it makes sense to state once up front that JSON is used. 
Thoughts?